src/test/java/support/fakes/FakeCQLToJSONInterpreter.java

```diff
@@ -76,10 +79,12 @@ private boolean containsSort(ImmutablePair<String, String> queryAndSort) {
 
     String splitRegex;
     BinaryOperator<Predicate<JsonObject>> accumulator;
-    //TODO Need to implement operator 'or' for query which contains search by different fields.
-    //This is just a temporary solution.
-    if (query.matches(PRECEDING_SUCCEEDING_URL_REGEX)) {
-      splitRegex = " or ";
+
+    final Matcher matcher = Pattern.compile(OR_REGEX)
```

Comment:
How about moving Pattern object to a field of the class? It seems to me we don't need to create pattern object every time when this method is called